src/test/java/com/akademia/projectplanner/service/impl/TaskDtoBuilder.java

```diff
@@ -0,0 +1,63 @@
+package com.akademia.projectplanner.service.impl;
+
+import com.akademia.projectplanner.dto.TaskDto;
+import com.akademia.projectplanner.enums.Status;
+
+import java.time.LocalDate;
+
+public class TaskDtoBuilder {
```

Comment:
ogólnie to nie trzeba pisać tej klasy, skoro projekt używa Lomboka to wystarczy dodać adnotację @Builder nad klasą TaskDTO, może zostać bo fajnie sobie przypomnieć jak to pisać ale ogólnie będzie to mała niekonsekwencja ;)